README.md

```diff
@@ -3,14 +3,41 @@
 [Pulumi Registry](https://pulumi.com/registry) is the global index of everything you can do with Pulumi.
 
 ## Authoring a Pulumi Package
+
 We’re always eager to expand that index with new [Pulumi Packages](https://www.pulumi.com/docs/guides/pulumi-packages/). Whether you want to author a new native provider, bridge a provider from the Terraform ecosystem, or create a cloud component with best practices and sensible defaults built in, we’d like to work with you to list it on Pulumi Registry.
 To get started, use our [guide on authoring a Pulumi Package](https://www.pulumi.com/docs/guides/pulumi-packages/how-to-author/) of your own.
 
-### Publishing a Package on the Registry
+### Publishing a Community Package on the Registry
+
+#### Community Steps
+
+To publish a community-maintained package on the Pulumi Registry as a community member:
+
+1. Ensure your provider repo has the following files:
+    * `docs/_index.md`, which should contain a summary of the provider's purpose (required) along with a code sample (preferred). This file will render as the index page for your provider ([example](https://www.pulumi.com/registry/packages/aiven/)).
+    * `docs/installation-configuration.md`, which should contain links to SDK packages in each language along with instructions for configuring the provider (e.g., required credentials and/or environment variables). This file will render as the Installation & Configuration page for your provider ([example](https://www.pulumi.com/registry/packages/aiven/installation-configuration/)).
+1. Create a release of your provider in GitHub.
+1. Add your package to the [community packages list](./community-packages/package-list.json) via pull request to this repository.
 
-When ready, add your package to the [community packages list](./community-packages/package-list.json) via pull request to this repository.
 For assistance, please reach out on the [Pulumi community slack](https://slack.pulumi.com/) or get in touch with us via this [contact form](https://pulumi.com/contact/?form=registry). 
 
+#### Pulumi Steps
+
+Once the community member has submitted the PR to add the provider to the registry, a Pulumi staff member should perform the following steps:
+
+1. Review the PR. Ensure that the PR has been rebased if necessary before merging. If ok, merge.
+1. Once the PR is merged, a [scheduled task](https://github.com/pulumi/registry/actions/workflows/generate-package-metadata.yml) will pick up the changes and create a PR to add the package metadata to the registry. A correct metadata PR ([example](https://github.com/pulumi/registry/pull/1606/files)) will include the following files, at a minimum:
```

Comment:
> at a minimum

This is only true when publishing the first release after adding the provider to the list. When the upstream author creates a new release of their provider, the automated PR might only contain the structured metadata file if the `docs` files were unchanged in between the releases.

Can you clarify this in the README to prevent confusion?

Reply:
Done. https://github.com/pulumi/registry/pull/1613